### DISCUSSION TITLE
Error while cargo install SUI Binnary

### DISCUSSION THREAD

**Discussion by Lord-Satchel** [MystenLabs/sui#5414] [Q&A]
I am getting this error while installing the SUI Binnary any ideas or suggestions ?
![image](https://user-images.githubusercontent.com/108660731/196971738-e23eb29d-cc91-4474-bd72-4ffa6b36a67d.png)

**Comment by mztacat** [comment]
Check for Rust compiler errors: Rust is a compiled language, so make sure that your code compiles without errors using the Rust compiler (rustc). Run rustc on your Rust source code to check for any compiler errors that may be causing the build script to fail.

**Comment by yanghelong1** [comment]
I'm having the same issue too
